hack/Dockerfile.in

```diff
@@ -27,6 +27,9 @@ RUN CGO_ENABLED=0 go build -o /buildroot/artifacts/{{COMPONENT}} cmd/*.go
 #
 # FINAL IMAGE
 #
+
+# gcr.io is deprecated, but gcr.io/distroless/static is hosted by artifact hub with the gcr.io path.
+# See: https://github.com/GoogleContainerTools/distroless/issues/1320
 FROM gcr.io/distroless/static:nonroot
```

Comment:
Not necessary to migrate this `gcr.io` image for reason in comment. It seems good to have this note. I would probably forget and then do all my google searching again in 6 months without it.